src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -266,10 +266,21 @@ private void WriteConvenienceMethod(LowLevelClientMethod clientMethod, Convenien
         {
             using (WriteConvenienceMethodDeclaration(_writer, convenienceMethod, fields, async))
             {
-                var contextVariable = new CodeWriterDeclaration(KnownParameters.RequestContext.Name);
-
-                var (parameterValues, converter) = convenienceMethod.GetParameterValues(contextVariable);
+                var requestBody = clientMethod.RequestMethod.Request.Body;
+                CodeWriterDeclaration? bodyOverrideVariable = null;
+                if (requestBody is RequestContentRequestBody body)
+                {
+                    var bodyInputParamter = clientMethod.RequestMethod.Operation.Parameters.First(p => p.Location == Common.Input.RequestLocation.Body);
+                    var bodyConvenienceParameter = convenienceMethod.ProtocolToConvenienceParameterConverters.First(c => c.Protocol.Name == KnownParameters.RequestContent.Name).Convenience;
+                    if (TypeFactory.IsDictionary(bodyConvenienceParameter.Type))
```

Comment:
Do we support nested dictionary, e.g. IReadOnlyDictionary<int, IReadOnlyDictionary<int, int>> in this solution?
or the element of the dictionary is another collection?

Reply:
Yes, we have tests "Dictionary_RecursiveModelValue_get" and "Dictionary_RecursiveModelValue_put".